performance/Makefile

```diff
@@ -0,0 +1,56 @@
+# Docker image
+IMAGE_REPO ?= determinedai/perf-test
+IMAGE_TAG ?= $(USER)
+
+# Perf testing cluster
+CLUSTER_ID ?= $(USER)-perf
+KEYPAIR_ID ?= $(USER)
+VERSION ?= $(shell det -v | cut -d' ' -f2)
+
+# Perf test options
+DET_ADMIN_USERNAME ?= admin
+DET_ADMIN_PASSWORD ?= ""
+
+IMAGE = $(IMAGE_REPO):$(IMAGE_TAG)
+DET_URL ?= $(shell cat .det.url || echo UNSET)
+
+build: Dockerfile src/* package-lock.json package.json tsconfig.json webpack.config.js
+	docker build . -t $(IMAGE)
+
+up:
+	det deploy aws up \
+		--cluster-id $(CLUSTER_ID) \
+		--keypair $(KEYPAIR_ID) \
+		--deployment-type simple-rds \
+		--det-version $(VERSION) \
+		--add-tag persist=false \
+		--yes \
+	| tee .up.out && \
+	cat .up.out \
+		| grep DET_MASTER \
+		| sed -rn 's/.*DET_MASTER=(.*)/\1/p' \
+		> .det.url
+
+down:
+	det deploy aws down --cluster-id $(CLUSTER_ID)
+
+run:
```

Comment:
my be nice to set ourselves up for lowest CI costs by adding a `run_local` target and recommending folks iterate on test writing with `run-local` and graduate to `run` when they need to actually test performance.

Reply:
Yeah that's a good point.
I'll try implementing it real quick using `det deploy local` and see how it goes.
I'll mention running `-local` targets in a README.